news/2017-11-28-OWC-Round-of-16-recap.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "osu! World Cup 2017 Round of 16 Recap"
+date: 2017-11-28 14:00:00 +0000
+---
+
+16 teams took to the stage over the weekend aiming to make it through to next weekend, and in fact, they all did! While the Round of 16 is the only stage that doesn't see any teams getting knocked out, there is still plenty on the line. 
+
+![](/wiki/shared/news/banners/owc_2017_recap_banner.jpg)
```

Comment:
banner missing from repo? did you clone branch from latest master after your last article was merged?